applications/serial_lte_modem/src/main.c

```diff
@@ -357,6 +358,75 @@ void handle_mcuboot_swap_ret(void)
 	slm_setting_fota_save();
 }
 
+int nidd_auto_connect(void)
+{
+	int err = 0;
+
+#if defined(CONFIG_LWM2M_CARRIER_SERVER_BINDING_N)
+	int cfun;
+
+	if (!fota_reconnect) {
+		return 0;
+	}
+	err = nrf_modem_at_scanf("AT+CFUN?", "+CFUN: %d", &cfun);
+	if (err != 1 || (cfun == 1 || cfun == 5)) {
+		return 0;
+	}
+
+	LOG_INF("NIDD auto connect");
+	err = nrf_modem_at_printf("AT%%XSYSTEMMODE=0,1,0,0");
```

Comment:
Two times %-character in the AT-command

Reply:
Double `%` is needed because `nrf_modem_at_printf()` takes a string fmt argument.